blockchain/types/receipt.go

```diff
@@ -101,6 +101,8 @@ type Receipt struct {
 	TxHash          common.Hash    `json:"transactionHash" gencodec:"required"`
 	ContractAddress common.Address `json:"contractAddress"`
 	GasUsed         uint64         `json:"gasUsed" gencodec:"required"`
+
+	CumulativeGasUsed uint64 `json:"cumulativeGasUsed"`
```

Comment:
Is this okay to add a field to `Receipt`? Shouldn't it be hard-forked to add a field of receipt?
If we are considering only API returns, how about adding the hard-coded field to API response format?

Reply:
I've been discussing this.
For now, We've decided to go the safer way, and I'll close this PR and modify the implementation to be counted every time the API is called.
Thank you for your review :)

Reply:
Ah, I agree with the way you talked. Thanks.